bundles/org.openhab.persistence.influxdb/pom.xml

```diff
@@ -1,47 +1,136 @@
+			<artifactId>influxdb-client-java</artifactId>
+			<version>1.6.0</version>
+		</dependency>
+		<dependency>
+			<artifactId>influxdb-client-core</artifactId>
+			<groupId>com.influxdb</groupId>
+			<version>1.6.0</version>
+		</dependency>
+		<dependency>
+			<artifactId>converter-gson</artifactId>
+			<groupId>com.squareup.retrofit2</groupId>
+			<version>2.5.0</version>
+		</dependency>
+		<dependency>
+			<artifactId>converter-scalars</artifactId>
+			<groupId>com.squareup.retrofit2</groupId>
+			<version>2.5.0</version>
+		</dependency>
+		<dependency>
+			<artifactId>gson</artifactId>
+			<groupId>com.google.code.gson</groupId>
+			<version>2.8.5</version>
+		</dependency>
+		<dependency>
+			<artifactId>gson-fire</artifactId>
+			<groupId>io.gsonfire</groupId>
+			<version>1.8.0</version>
+		</dependency>
+		<dependency>
+			<artifactId>okio</artifactId>
+			<groupId>com.squareup.okio</groupId>
+			<version>1.17.3</version>
+		</dependency>
+		<dependency>
+			<artifactId>commons-csv</artifactId>
+			<groupId>org.apache.commons</groupId>
+			<version>1.6</version>
+		</dependency>
+		<dependency>
+			<artifactId>json</artifactId>
+			<groupId>org.json</groupId>
+			<version>20180813</version>
+		</dependency>
+		<dependency>
+			<artifactId>okhttp</artifactId>
+			<groupId>com.squareup.okhttp3</groupId>
+			<version>3.13.1</version>
+		</dependency>
+		<dependency>
+			<artifactId>retrofit</artifactId>
+			<groupId>com.squareup.retrofit2</groupId>
+			<version>2.5.0</version>
+		</dependency>
+		<dependency>
+			<artifactId>jsr305</artifactId>
+			<groupId>com.google.code.findbugs</groupId>
+			<version>3.0.2</version>
+		</dependency>
+		<dependency>
+			<artifactId>logging-interceptor</artifactId>
+			<groupId>com.squareup.okhttp3</groupId>
+			<version>3.13.1</version>
+		</dependency>
+		<dependency>
+			<artifactId>rxjava</artifactId>
+			<groupId>io.reactivex.rxjava2</groupId>
+			<version>2.2.17</version>
+		</dependency>
+		<dependency>
+			<artifactId>reactive-streams</artifactId>
+			<groupId>org.reactivestreams</groupId>
+			<version>1.0.3</version>
+		</dependency>
+		<dependency>
+			<artifactId>swagger-annotations</artifactId>
+			<groupId>io.swagger</groupId>
+			<version>1.5.22</version>
+		</dependency>
+		<!--END influxdb-client-java-->
+
+
+		<dependency>
+			<groupId>com.influxdb</groupId>
+			<artifactId>flux-dsl</artifactId>
+			<version>1.6.0</version>
+		</dependency>
+
+		<!--END InfluxDB 2.0-->
+
+		<!--START InfluxDB 1.0-->
+		<dependency>
+			<groupId>org.influxdb</groupId>
+			<artifactId>influxdb-java</artifactId>
+			<version>2.2</version>
+		</dependency>
+		<dependency>
+			<groupId>com.squareup.okhttp</groupId>
+			<artifactId>okhttp</artifactId>
+			<version>2.4.0</version>
+		</dependency>
+		<dependency>
+			<groupId>com.squareup.retrofit</groupId>
+			<artifactId>retrofit</artifactId>
+			<version>1.9.0</version>
+		</dependency>
+		<!-- END InfluxDB 1.0 -->
+	</dependencies>
```

Comment:
Unless formatting standards have changed, the pom file is supposed to have 2 space indentation, not tabs. If this is new with oh3 then please correct me.

Reply:
I'm really a newbie. But I think that I changed to tabs because one of the static code analysis said it doesn't like spaces,...

Reply:
yea, I'm not a fan of that requirement either. But the tab requirement is only for the other xml files, not the pom file.